server/testutils/containere2e/containere2e.go

```diff
@@ -0,0 +1,77 @@
+package containere2e
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"path/filepath"
+	"testing"
+
+	"github.com/mattermost/mattermost-plugin-msteams-sync/server/store/sqlstore"
+	"github.com/mattermost/mattermost-plugin-msteams-sync/server/testutils/mmcontainer"
+	"github.com/mattermost/mattermost-plugin-msteams-sync/server/testutils/testmodels"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/stretchr/testify/require"
+)
+
+func NewE2ETestPlugin(t *testing.T) (*mmcontainer.MattermostContainer, *sqlstore.SQLStore, func()) {
+	ctx := context.Background()
+	matches, err := filepath.Glob("../../dist/*.tar.gz")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(matches) == 0 {
+		t.Fatal("Unable to find plugin tar.gz file")
+	}
+	filename := matches[0]
+
+	pluginConfig := map[string]any{
+		"clientid":                   "client-id",
+		"clientsecret":               "client-secret",
+		"connectedusersallowed":      1000,
+		"encryptionkey":              "eyPBz0mBhwfGGwce9hp4TWaYzgY7MdIB",
```

Comment:
This looks "real" -- I'm guessing it's just random? Should we replace it with something like `encryption-key` or would that break things?

Reply:
Exactly, I need an "encryption key" that is valid.